shoop/simple_supplier/utils.py

```diff
@@ -31,3 +49,68 @@ def get_current_stock_value(supplier_id, product_id):
         "logical_count": events - orders_bought,
         "physical_count": events - orders_sent
     }
+
+
+def get_stock_information_div_id(supplier, product):
+    return "stock-information-%s-%s" % (supplier.id, product.id)
+
+
+def get_stock_information_html(supplier, product):
+    """
+    Get html string to show current stock information for product
+
+    :param supplier: shoop Supplier
+    :type supplier: shoop.core.models.Supplier
+    :param product: shoop Product
+    :type product: shoop.core.models.Product
+    :return: html div as a sting
+    :rtype: str
+    """
+    stock = StockCount.objects.filter(product=product, supplier=supplier).first()
+    context = {
+        "div_id": get_stock_information_div_id(supplier, product),
+        "sales_decimals": product.sales_unit.decimals if product.sales_unit else 0,
+        "sales_unit": product.sales_unit.short_name if product.sales_unit else "",
+        "stock": stock
+    }
+    return render_to_string("shoop/simple_supplier/admin/stock_information.jinja", context)
+
+
+def get_stock_adjustment_div(request, supplier, product):
+    """
+    Get html string to adjust stock values
+
+    Contains inputs for purchase_price_value and delta
+
+    :param request: HTTP request
+    :type request: django.http.HttpRequest
+    :param supplier: shoop Supplier
+    :type supplier: shoop.core.models.Supplier
+    :param product: shoop Product
+    :type product: shoop.core.models.Product
+    :return: html div as a sting
+    :rtype: str
+    """
+    latest_adjustment = StockAdjustment.objects.filter(
+        product=product, supplier=supplier).order_by("-created_on").first()
+    context = {
+        "csrf_token": request.COOKIES.get("csrftoken"),
+        "product": product,
+        "supplier": supplier,
+        "purchase_price_input_name": get_purchase_price_input_name(),
+        "purchase_price_label": "%s [%s]" % (_("Purchase price per unit"), SHOOP_HOME_CURRENCY),
```

Comment:
This should probably use the i18n api to localize the currency name (or at least fetch the currency symbol).